src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -316,6 +316,43 @@ internal static SemanticVersion PSV6Version
     }
 
     /// <summary>
+    /// Represents an implementation of '$PSVersionTable' variable.
+    /// The implementation contains ordered 'Keys' and 'GetEnumerator' to get user-friendly output.
+    /// </summary>
+    public sealed class PSVersionHashTable : Hashtable, IEnumerable
+    {
+        internal PSVersionHashTable(IEqualityComparer equalityComparer) : base(equalityComparer)
+        {
+        }
+
+        /// <summary>
+        /// Returns ordered collection with Keys of 'PSVersionHashTable'
+        /// </summary>
+        public override ICollection Keys
+        {
+            get
+            {
+                Array arr = new string[base.Keys.Count];
+                base.Keys.CopyTo(arr, 0);
+                Array.Sort(arr, StringComparer.OrdinalIgnoreCase);
```

Comment:
Agree 100%.  About the **only** thing I ever look for from this info is `PSVersion`.  This will make it harder to see that version.